keys.go

```diff
@@ -111,6 +111,21 @@ func NewSecretKey(key [SecretKeySize]byte, nonce [32]byte) *SecretKey {
 	}
 }
 
+func NewSecretKeyFromEd25519Bytes(b [SecretKeySize + 32]byte) *SecretKey {
+	sk := &SecretKey{
+		key:   [SecretKeySize]byte{},
+		nonce: [32]byte{},
+	}
+
+	copy(sk.key[:], b[:32])
+	t := divideScalarByCofactor(sk.key[:])
+
+	copy(sk.key[:], t)
```

Comment:
```suggestion
	copy(sk.key[:], divideScalarByCofactor(b[:32]))
```

Reply:
This suggested change is not correct, because `divideScalarByCofactor` actually changes the slice in argument. But I'll commit the following, that has one `copy` less. Thanks.

```
diff --git a/keys.go b/keys.go
index 4ebfe1d..da0cf5b 100644
--- a/keys.go
+++ b/keys.go
@@ -118,9 +118,8 @@ func NewSecretKeyFromEd25519Bytes(b [SecretKeySize + 32]byte) *SecretKey {
        }
 
        copy(sk.key[:], b[:32])
-       t := divideScalarByCofactor(sk.key[:])
+       divideScalarByCofactor(sk.key[:])
 
-       copy(sk.key[:], t)
        copy(sk.nonce[:], b[32:])
 
        return sk
```